pom.xml

```diff
@@ -6,7 +6,7 @@
 
   <groupId>com.intuit.graphql</groupId>
   <artifactId>graphql-filter-java</artifactId>
-  <version>1.0.0-SNAPSHOT</version>
```

Comment:
Can you please revert this changes, as it will be taken care during the release process.

Reply:
Just reverted, thank you!